src/esssans/normalization.py

```diff
@@ -70,7 +70,7 @@ def solid_angle(
     length = sc.norm(cylinder_axis)
 
     omega = _approximate_solid_angle_for_cylinder_shaped_pixel_of_detector(
-        pixel_position=data.coords['position'],
+        pixel_position=data.coords['position'] - data.coords['sample_position'],
```

Comment:
Do we have a unit test with a sample that is not at the origin?

Reply:
No in that case I think that test should have failed in this PR

Reply:
Yes we do, I was confused. The current test has a sample not at the origin (its at [0, 0, 19.28]).

The issue was that I thought the `pixel_positions` from the dataarray were relative to the sample, and therefore I didn't subtract the `sample_position` in the provider.
Therefore the tests were made from that assumption, and the data was prepared accordingly, and that's why the tests did not fail before.

Reply:
Where did the data change in this PR?

Reply:
Ah, I see, you computed the difference in the test setup?

Reply:
Yes exactly.